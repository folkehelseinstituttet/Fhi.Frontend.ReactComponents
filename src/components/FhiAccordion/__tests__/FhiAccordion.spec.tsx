import renderer from 'react-test-renderer';
import Accordion from '../FhiAccordion';

const DefaultHeader = () => <h4>Dette er en ustylet overskrift</h4>;
const DefaultContent = () => <div><span>Dette er ustylet innhold</span></div>;

const defaultArgs = {
  items: [
    {
      id: '1',
      header: <DefaultHeader />,
      content: <DefaultContent />,
    },
    {
      id: '2',
      header: <DefaultHeader />,
      content: <DefaultContent />,
    },
  ],
  className: 'Testclass',
};

describe('FhiAccordion', () => {
  test('snapshot test', () => {
    const accordion = renderer.create(
      <Accordion
        className="test"
        items={defaultArgs.items}
      />,
    );
    const tree = accordion.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
