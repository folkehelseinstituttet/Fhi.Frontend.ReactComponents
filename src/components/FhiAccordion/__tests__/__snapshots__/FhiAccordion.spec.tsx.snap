// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FhiAccordion snapshot test 1`] = `
<div>
  <div
    class="test  accordion"
  >
    <div
      class="accordion-item"
    >
      <h2
        class="accordion-header"
      >
        <button
          aria-expanded="false"
          class="accordion-button collapsed"
          type="button"
        >
          <h4>
            Dette er en ustylet overskrift
          </h4>
        </button>
      </h2>
      <div
        class="accordion-collapse collapse"
      >
        <div
          class="accordion-body"
        >
          <div>
            <span>
              Dette er ustylet innhold
            </span>
          </div>
        </div>
      </div>
    </div>
    <div
      class="accordion-item"
    >
      <h2
        class="accordion-header"
      >
        <button
          aria-expanded="false"
          class="accordion-button collapsed"
          type="button"
        >
          <h4>
            Dette er en ustylet overskrift
          </h4>
        </button>
      </h2>
      <div
        class="accordion-collapse collapse"
      >
        <div
          class="accordion-body"
        >
          <div>
            <span>
              Dette er ustylet innhold
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
